functests/setup_teardown.go

```diff
@@ -23,47 +22,35 @@ func BeforeTestSuiteSetup() {
 
 	err = t.StartDefaultStorageCluster()
 	gomega.Expect(err).To(gomega.BeNil())
-
 }
 
 // AfterTestSuiteCleanup is the function called to tear down the test environment
 func AfterTestSuiteCleanup() {
 	flag.Parse()
-
 	t, err := deploymanager.NewDeployManager()
 	gomega.Expect(err).To(gomega.BeNil())
 
 	err = t.DeleteNamespaceAndWait(TestNamespace)
 	gomega.Expect(err).To(gomega.BeNil())
 
-	// TODO uninstall storage cluster.
-	// Right now uninstall doesn't work. Once uninstall functions
-	// properly, we'll want to uninstall the storage cluster after
-	// the testsuite completes
+	err = t.UninstallOCS(OcsRegistryImage, LocalStorageRegistryImage, OcsSubscriptionChannel)
+	gomega.Expect(err).To(gomega.BeNil())
```

Comment:
The comment that this uninstall replaces says
> Right now, uninstall doesn't work. Once uninstall functions properly, we'll want to uninstall the storage after the testsuite completes

The test is currently failing at this stage. So is uninstall functioning properly yet? Your commit message says nothing about that. And the PR description does not even mention uninstall, so this sounds suspicious...